src/components/DatePicker/datepickerPropTypes.js

```diff
@@ -18,11 +18,16 @@ const propTypes = {
      * `onInputChange` would always be called with a Date (or null)
      */
     defaultValue: PropTypes.oneOfType([PropTypes.instanceOf(Date), PropTypes.string]),
+
+    minDate: PropTypes.oneOfType([PropTypes.instanceOf(Date), PropTypes.string]),
```

Comment:
I suggest unifying the types to a single type, Date instead of a Date or a string